content/docs/user-guide/gems/reference/aws/aws-gamelift/cpp-api.md

```diff
@@ -130,22 +129,27 @@ As the default behavior, when the last player leaves the game session, GameLift
 
 ## Server notifications
 
-The AWS GameLift Gem notifies the GameLift service that your server process is ready to host game sessions, handle requests, and take connections.
+You must notify Amazon GameLift service that your server process is ready to host game sessions, handle requests, and take connections.
+
+To send a notification that your server process is ready, use `AWSGameLiftServerRequestBus::Events::NotifyGameLiftProcessReady()`.
+Put this call in a proper place on your server side, like server system component activate step, and call it after you connect your project to `AzFramework::SessionNotificationBus`.
```

Comment:
nit: "proper place" seems to indicate there is a definite place. Maybe "Call process ready once your have completed any relevant initialization. One recommended sport is in the [ServerSystemCompontent]'s activate step, after connecting to the `AzFramework::SessionNotificationBus`